protofire/omen-exchange
# Fix/#1412

closes https://github.com/protofire/omen-exchange/issues/1412 

Adds support for gs_multi_send and removes safe-apps-sdk as a dependency.

To test: 

1. Reinstall node modules
2. Run Omen locally
3. Go to https://xdai.gnosis-safe.io/app/#/welcome or https://rinkeby.gnosis-safe.io/app/#/welcome and setup a safe if you haven't yet
4. Go to Apps in the left sidebar
5. Select the Wallet Connect app
6. To make sure the wallet connect app you are using is the correct one, make sure the url has the following at the end: `appUrl=https://ipfs.io/ipfs/QmT3VxxfFtfEcvq8AeaoFAyUGxePRa2zisvnxLTrQXU5Uf`
7. In Omen, log in with wallet connect - screen shot the QR code, select the screenshot file and copy it
8. Back in the Wallet Connect app, paste the QR code screen shot
9. Now you should be logged in. When you make transactions on Omen you should be able to confirm transactions from the wallet connect app
